binding.gyp

```diff
@@ -90,17 +87,11 @@
       'include'
     ],
     'defines': [
-      'GPR_BACKWARDS_COMPATIBILITY_MODE'
+      'GPR_BACKWARDS_COMPATIBILITY_MODE',
+      'GRPC_ARES=0',
+      'GRPC_UV'
```

Comment:
related to below comment: with all of the `ifdef GRPC_UV`'s removed, can this be taken out?